assets/aml-benchmark/components/prompt_crafter/spec.yaml

```diff
@@ -93,6 +93,11 @@ inputs:
     optional: true
     default: 0
     description: Random seed for sampling few-shots; if not specified, 0 is used.
+  ground_truth_column_name:
+    type: string
+    optional: true
+    default: ''
+    description: Label key/ ground truth column to be passed in the output.
```

Comment:
Can we remove `Label key/` from here? Also, can we update the description to explain why this is optional and when to use it?

Reply:
Updated.